myhpi/static/scss/myHPI.scss

```diff
@@ -262,4 +262,11 @@ h1.toc-title::after {
         margin-top: 50px;
         order: 3;
     }
+}
+
+.nav-bar-group-header {
+    padding: $dropdown-item-padding-y $dropdown-item-padding-x;
+}
+.nav-bar-group-item {
+    padding: $dropdown-item-padding-y $dropdown-item-padding-x*2;

```

Comment:
Nav-related stuff can go into `navbar.scscc`

Reply:
As of right now, only myHPI.scss appears to be imported into the base template. 
```$dropdown-item-padding``` is used in the bootstrap templates for dropdown items. 
Thus far i have found no occurrence of the class being overridden. Hence, ```$dropdown-item-padding```  is used (and results equal spacing across the different dropdown-items). 

Nevertheless, I will investigate to ensure proper behaviour :).